src/components/Body.js

```diff
@@ -6,6 +6,13 @@ const Body = props => {
     <div className="bg-light">
```

Comment:
The links are all showing on one line with no spaces between:
![image](https://user-images.githubusercontent.com/31415142/47605029-08389000-d9b6-11e8-962a-cfbf7416a0cd.png)

Try adding a link style like 
``` javascript
const linkStyle = {
  display: "block"
}

```